src/tools/md-to-md-migrator.md

```diff
@@ -0,0 +1,30 @@
+---
+title: MangaDex backup migrator
+lang: en-US
+---
+
+:::: guide MangaDex to MangaDex backup migrator
+MangaDex changed their ids to UUIDS. If the source remains compatible with legacy ids, using them will be slower as it requires to call a conversion endpoint every time.
+
+To migrate you library:
```

Comment:
Your*

Reply:
Thank you for your review Ankah! Fixed by 11223d8